types/src/transaction/authenticator.rs

```diff
@@ -907,6 +920,12 @@ impl AnySignature {
             (Self::Secp256k1Ecdsa { signature }, AnyPublicKey::Secp256k1Ecdsa { public_key }) => {
                 signature.verify(message, public_key)
             },
+            (
+                Self::WebAuthn {
+                    partial_authenticator_assertion_response,
+                },
+                _,
```

Comment:
nit: this looks confusing, just rebind public_key here is more readable

Reply:
For context, right now we're only supporting `P256` with WebAuthn but in the future one could add Ed25519 and other signature schemes with WebAuthn. This is why I chose not to rebind in this location. In the verification for `PartialAuthenticatorAssertionResponse` I check to make sure it's an `AnyPublicKey::P256Ecdsa`. Lmk if this makes sense or if you think I should rebind anyways. @zekun000